app/views/product_users/index.html.haml

```diff
@@ -20,6 +20,12 @@
   %p.notice= text("none")
 - else
   - if @product.has_product_access_groups?
+    = form_tag facility_instrument_users_path(format: :csv), method: :get, class: "search_form" do
```

Comment:
I'm confused...where's the action button for this form? Appears to be related to the link_to...hmm..do we need the form? can it be a button only?

Reply:
This is the mechanism that other Export to CSV links in the app are using. They have a form with hidden fields that a javascript snippet populates through a pop-up input box and that's how we are able to pass the input email on to the backend through an xhr call. I was duplicating this functionality (though I agree it is not ideal)